src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -108,21 +108,25 @@ class TezosNodeOperator(node: TezosRPCInterface) extends LazyLogging {
     * @param hash     Hash of the given block
     * @return         Block
     */
-  def getBlock(network: String, hash: String): Try[TezosTypes.Block] =
-    node.runGetQuery(network, s"blocks/$hash").flatMap { jsonEncodedBlock =>
+  def getBlock(network: String, hash: String, offset: Option[Int] = None): Try[TezosTypes.Block] = {
+    val offsetString = offset.map(_.toString).getOrElse("")
+    val offsetStringWithTilde = offsetString match {
+      case "" => ""
+      case offset => "~" + offset
+    }
+    node.runGetQuery(network, s"blocks/$hash${offsetStringWithTilde}").flatMap { jsonEncodedBlock =>
```

Comment:
no need for the curlies if the interpolated value is a simple name

`s"block/$hash$offsetWithTilde"` is ok